### PR TITLE
implement achievements

### PR DESCRIPTION
**Pull request by mailtime-wing**

### CODE REVIEW

File: src/components/GatsbyImage/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from "react"
```
Comment (chochihim) [inline src/components/GatsbyImage/index.js]: Please remove unused components

File: src/components/LogosSection/Achievements/index.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import {
+  achievementsImage,
+  imageContainer,
+  andMore,
+  inlineContainer,
+  inlineContainer1,
+} from "./style"
+
+const Achievements = () => {
+  const Image = ({ fluid }) => <Img css={achievementsImage} fluid={fluid} />
```
Comment (chochihim) [inline src/components/LogosSection/Achievements/index.js]: Please put this outside of `<Achievements />`

File: src/components/LogosSection/BackedBy/index.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+import {
+  imageContainer,
+  backedByImage,
+  inlineContainer,
+  inlineContainer1,
+  andMore,
+} from "./style"
+
+const BackedBy = () => {
+  const Image = ({ fluid }) => <Img css={backedByImage} fluid={fluid} />
```
Comment (chochihim) [inline src/components/LogosSection/BackedBy/index.js]: Please put this outside of `<BackedBy />`

File: src/components/LogosSection/TrustedBy/index.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import { FormattedMessage } from "gatsby-plugin-intl"
+import { useStaticQuery, graphql } from "gatsby"
+import Img from "gatsby-image"
+
+import { trustedByImage, andMore } from "./style"
+
+const TrustedBy = () => {
+  const Image = ({ fluid }) => <Img css={trustedByImage} fluid={fluid} />
```
Comment (chochihim) [inline src/components/LogosSection/TrustedBy/index.js]: Please put this outside of `<BackedBy />`

File: src/components/GatsbyImage/index.js (hunk 1 above)
Comment (mailtime-wing) [reply src/components/GatsbyImage/index.js]: removed.

File: src/components/LogosSection/Achievements/index.js (hunk 2 above)
Comment (mailtime-wing) [reply src/components/LogosSection/Achievements/index.js]: done.

File: src/components/LogosSection/BackedBy/index.js (hunk 3 above)
Comment (mailtime-wing) [reply src/components/LogosSection/BackedBy/index.js]: done.

File: src/components/LogosSection/TrustedBy/index.js (hunk 4 above)
Comment (mailtime-wing) [reply src/components/LogosSection/TrustedBy/index.js]: done.

**Pull request merged**